src/base/bittorrent/torrenthandle.h

```diff
@@ -41,10 +41,8 @@
 #include <QVector>
 
 #include <libtorrent/torrent_handle.hpp>
-#include <libtorrent/version.hpp>
-#if LIBTORRENT_VERSION_NUM >= 10100
 #include <libtorrent/torrent_status.hpp>
-#endif
+#include <libtorrent/version.hpp>
```

Comment:
This is not used anymore.

Reply:
Same in torrentinfo.h trackerentry.h and everywhere else, you might want to remove all libtorrent/version.hpp if possible.

Reply:
Ok. Fixed.